### PR TITLE
ApplePay request session

### PR DESCRIPTION
**Pull request by sandervanhooft**

### CODE REVIEW

**Comment by sandervanhooft** [comment]
Still wrapping my head around this integration.

Based on the Mollie specs, this is my best guess.

**Comment by sandervanhooft** [comment]
- [ ] Update the client version attribute before releasing.

File: tests/Mollie/API/Endpoints/WalletEndpointTest.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                $responseBody
+            )
+        );
+
+        $domain = 'pay.mywebshop.com';
+        $validationUrl = 'https://apple-pay-gateway-cert.apple.com/paymentservices/paymentSession';
+
+        $response = $this->apiClient->wallets->requestApplePayPaymentSession($domain, $validationUrl);
+
+        $this->assertEquals(
```
Comment (willemstuursma) [inline tests/Mollie/API/Endpoints/WalletEndpointTest.php]: asssertJsonStringEqualsJsonString

Comment (sandervanhooft) [reply tests/Mollie/API/Endpoints/WalletEndpointTest.php]: Good one. Updated

Comment (sandervanhooft) [reply tests/Mollie/API/Endpoints/WalletEndpointTest.php]: Good one. Updated.

**Comment by sandervanhooft** [comment]
I don't see any impact on the Create Payment call (`applePayPaymentToken`).

Unless the applePayPaymentToken attribute is included in the payment response (Payment Resource). 

Correct, @willemstuursma ?

File: src/Endpoints/WalletEndpoint.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        // Not used
+    }
+
+    public function requestApplePayPaymentSession($domain, $validationUrl)
+    {
+        $response = $this->client->performHttpCall(
+            self::REST_CREATE,
+            'wallets/applepay/sessions',
+            $this->parseRequestBody([
+
```
Comment (LauLaman) [inline src/Endpoints/WalletEndpoint.php]: You should pass on the parameters:
```
[
    'domain'=> $domain,
    'validationUrl' => $validationUrl,
]
```

Comment (sandervanhooft) [reply src/Endpoints/WalletEndpoint.php]: You're right! 😮

**Comment by sandervanhooft** [comment]
Passing on parameters now (thanks @LauLaman !). But now running into issues with `mockApiCall`.

**Comment by sandervanhooft** [comment]
Fixed the test.

**Comment by sandervanhooft** [comment]
@LauLaman Fixed it. Can you give it a spin?

**Comment by sandervanhooft** [comment]
@willemstuursma What do you think, ready for release?

**Pull request merged**